_projects/optimize_caching.md

```diff
@@ -40,10 +43,12 @@ this greatly improved coala's startup speed.
 Performance is key to usability - with faster runtimes, users can get on
 with their work quicker. And it's fun looking for performance bottlenecks!
 
-In this project, you're to improve on the current strategies. Some enhancements
-such as section-level caching, globbing improvements to not expand on
-unmodified directories, and others have been preliminarily explored. Please
-refer to the related issues to learn more about these.
+In this project, you're to come up with improved versions of the current caching
+strategies used by coala's core and integrate these with the NextGen-Core.
+Some enhancements such as ignoring unmodified directories and use of
+`FileFactory` and `Directory` classes have been already explored and are to be
```

Comment:
These changes depend on the cEP being accepted.

Reply:
@palash25 you can link to the old version of cEP-0002 that still contains the `FileProxy`, then you avoid describing features ahead of this cEP.